pkg/tests/apis/shorturl/shorturl_test.go

```diff
@@ -211,6 +211,7 @@ func doDualWriteTests(t *testing.T, helper *apis.K8sTestHelper, mode grafanarest
 		found, err := client.Resource.Get(context.Background(), uid, metav1.GetOptions{})
 		require.NoError(t, err)
 		assert.Equal(t, uid, found.GetName())
+		assert.LessOrEqual(t, time.Since(found.GetCreationTimestamp().Time).Seconds(), 30.0, "creation timestamp should be within last 30 seconds")
```

Comment:
This line would have caught the issue. The 30-second value is just long enough to avoid adding a flaky test. The important thing is to check that creationTimestamp has a reasonable value.